integration_tests/src/util/workflows.rs

```diff
@@ -181,18 +191,21 @@ pub fn miner_prove_sector(
     ExpectInvocation {
         from: worker_id,
         to: *miner_id,
-        method: MinerMethod::ProveCommitSector as u64,
-        subinvocs: Some(vec![ExpectInvocation {
-            from: miner_id.id().unwrap(),
-            to: STORAGE_POWER_ACTOR_ADDR,
-            method: PowerMethod::SubmitPoRepForBulkVerify as u64,
-            ..Default::default()
-        }]),
+        method: MinerMethod::ProveCommitSectors3 as u64,
         ..Default::default()
     }
     .matches(v.take_invocations().last().unwrap());
 }
 
+pub fn miner_prove_cc_sector(
```

Comment:
is it worth having this for a single use? empty deal vec ought to be fairly obvious I think

Reply:
You're right.  Deleted.